examples/with-firebase-cloud-messaging/static/firebase-messaging-sw.js

```diff
@@ -0,0 +1,8 @@
+importScripts('https://www.gstatic.com/firebasejs/4.8.1/firebase-app.js')
+importScripts('https://www.gstatic.com/firebasejs/4.8.1/firebase-messaging.js')
+
+firebase.initializeApp({
+	messagingSenderId: 'your sender id'
+})
+
+firebase.messaging()
```

Comment:
Looks good perhaps add this so that you know the service worker was installed correct and so messages show up in the background 

```
self.addEventListener('install', (event) => {
  console.log('Service worker installed');
});


messaging.setBackgroundMessageHandler(function(payload) {
  console.log('[firebase-messaging-sw.js] Received background message ', payload);
  // Customize notification here
  var notificationTitle = 'Background Message Title';
  var notificationOptions = {
    body: 'Background Message body.',
    icon: '/firebase-logo.png'
  };

  return self.registration.showNotification(notificationTitle,
    notificationOptions);
  });
```